### ISSUE TITLE
Have a web section for the OCaml Paris meetups, past and future

### ISSUE THREAD

**Issue by gasche** [ocaml/v2.ocaml.org#148]
We have had two local meetups in Paris so far, and it would be nice to have webpages to put the slides of the past talks, and maybe announce the next ones in a central place. I think ocaml.org would be a relevant place for this -- of course, it could also host webpages for other local events in the world.

What do you think on the principle of having sections for local event?
If that's accepted, any advice on the structure? Where in the internal hierarchy, and which existing page should I reuse as a template?

**Comment by lefessan** [comment]
It would be nice to have a calendar too, for such events, on the front page, different from the calendar for "big events" (like OCaml'2013), and different from the calendar of package releases. It would also show that the community is active.

**Comment by avsm** [comment]
I agree that having a central place to link to all the writeups of meetups would be good (including the Cambridge ones, or the big OCaml meetups).  I'm not so sure about hosting the big slides and videos on this site though, as that'll make it quite large.

What do you think about storing the slides on a third-party service like Slideshare and Youtube?  We do that for (e.g.) CUFP, and it covers the big content reasonably well.  On the other hand, these services come and go all the time, and we do longer-term archival is useful.

**Comment by gasche** [comment]
I have a strong dislike for Slideshare in particular as a slide provider (crappy interface, hard to find how to download), and rarely produce slides that use more than 500Kib of disk space. I'm not sure what count as "large" these days, but I'm a bit surprised slides would be the problem. Maybe with large illustrations, etc. If the only problem is space, I would propose you set some arbitrary soft limit, and we discuss what to do with outsiders when they appear.

**Comment by avsm** [comment]
The rule of thumb is essentially what fits into Git:
https://help.github.com/articles/what-is-my-disk-quota

Once you put something into this repository, it can't easily be taken out without rebasing the entire repository and invalidating all the forks.

We could easily sidestep this by creating a ocaml.org-media repository and dumping the binaries in there.  They'll largely be immutable, and we could rebase that repository if the space use ever became a problem.  This will cover PDFs and slides without too many problems.

Either way, videos definitely need to be hosted externally.  Is there a less crappy provider than Slideshare to mirror some of these slides to?  I'd like to find something with an API.  Bonus points if the slides and the videos could also be viewed together!

**Comment by gasche** [comment]
> We could easily sidestep this by creating a ocaml.org-media repository and dumping the binaries in there. They'll largely be immutable, and we could rebase that repository if the space use ever became a problem. This will cover PDFs and slides without too many problems.

That seems like a reasonable choice. Should I create the repo, or will you (or anyone in the ocaml.org organization) take care of it?

Regarding videos, I'm personally fine with any kind of hosting (eg. Youtube): videos already are a bad format to consume content, so the interface is not going to make a difference. Videos are also not the responsibility of the presenter themselves, while they should be contacted to get the source material of the presentations.

**Comment by agarwal** [comment]
On Mon, Jun 3, 2013 at 9:57 AM, gasche notifications@github.com wrote:

>  That seems like a reasonable choice. Should I create the repo, or will
> you (or anyone in the ocaml.org organization) take care of it?
> 
> It already exists [1], but we hadn't started using it yet. Feel free to
> start making pull requests. We can optimize the directory structure once
> some media is added, and it becomes more clear what we want in there.

[1] https://github.com/ocaml/ocaml.org-media

In general the issue is whether the ocaml.org project is taking
responsibility for the content. This depends on the manner in which it is
being presented. If we create a Videos page, add headers to the videos,
etc., then that page's structure depends on the videos existing. Thus, the
videos should be under the ocaml.org project's control, which can be
accomplished in two ways: we host them or use the "ocamlwebsite" youtube
account.

On the other hand, maybe we just have a list of videos related to OCaml.
It's not a big deal if some items in the list become obsolete or disappear.
In this case, we can just link to URL's not necessarily under our control.

**Comment by amirmc** [comment]
There is an open issue on how to deal with user groups at #97. My view is that ocaml.org shouldn't be hosting such user group pages/content directly.  I'd be more inclined for user groups to have their own online presence and link to those sites (with short blurbs) from a community section of the site.  It gives the groups freedom to do whatever they want with their site while still being discoverable from the overall language site and doesn't burden ocaml.org with the maintenance of (nor features for) user group pages.

Of course, any _write-ups_ of the events would be great to include through the ocaml-planet feed.

Regarding hosting of files, you can consider speakerdeck (I think it looks nice) [1] and videos could also go on vimeo [2] instead of youtube.  A calendar of all events happening around the world would be great but it should work the same way as a feed (i.e people just submit their webcal feed of events, the same way people add their blogs to planet).

[1] https://speakerdeck.com/
[2] https://vimeo.com

**Comment by Chris00** [comment]
@gasche Are you still interested in adding pages for the Paris meeting — hosting the slides on, say, https://github.com/ocaml/ocaml.org-media ?  the meetings are listed at http://ocaml.org/meetings/ — the slides would be a nice addition IMHO.
